ml_metadata/tools/mlmd_bench/fill_context_edges_workload_test.cc

```diff
@@ -0,0 +1,295 @@
+          context_node_popularity: {dirichlet_alpha : 1}
+        }
+      )";
+
+// Enumerates the workload configurations as the test parameters that ensure
+// test coverage.
+std::vector<WorkloadConfig> EnumerateConfigs() {
+  std::vector<WorkloadConfig> configs;
+
+  {
+    WorkloadConfig config =
+        testing::ParseTextProtoOrDie<WorkloadConfig>(config_str);
+    config.set_num_operations(kNumberOfOperations);
+    config.mutable_fill_context_edges_config()
+        ->mutable_num_edges()
+        ->set_minimum(kNumberOfEdgesPerRequest);
+    config.mutable_fill_context_edges_config()
+        ->mutable_num_edges()
+        ->set_maximum(kNumberOfEdgesPerRequest);
+    config.mutable_fill_context_edges_config()->set_specification(
+        FillContextEdgesConfig::ATTRIBUTION);
+    configs.push_back(config);
+  }
+
+  {
+    WorkloadConfig config =
+        testing::ParseTextProtoOrDie<WorkloadConfig>(config_str);
+    config.set_num_operations(kNumberOfOperations);
+    config.mutable_fill_context_edges_config()
+        ->mutable_num_edges()
+        ->set_minimum(kNumberOfEdgesPerRequest);
+    config.mutable_fill_context_edges_config()
+        ->mutable_num_edges()
+        ->set_maximum(kNumberOfEdgesPerRequest);
+    config.mutable_fill_context_edges_config()->set_specification(
+        FillContextEdgesConfig::ASSOCIATION);
+    configs.push_back(config);
+  }
+
+  return configs;
+}
+
+// Gets the number of existed context edges in db. Returns detailed error
+// if query executions failed.
+tensorflow::Status GetNumberOfContextEdgesInDb(
+    const FillContextEdgesConfig& fill_context_edges_config,
+    MetadataStore& store, int64& num_context_edges) {
+  GetContextsResponse get_response;
+  TF_RETURN_IF_ERROR(store.GetContexts(
+      /*request=*/{}, &get_response));
+  for (const auto& context : get_response.contexts()) {
+    switch (fill_context_edges_config.specification()) {
+      case FillContextEdgesConfig::ATTRIBUTION: {
+        GetArtifactsByContextRequest request;
+        request.set_context_id(context.id());
+        GetArtifactsByContextResponse response;
+        TF_RETURN_IF_ERROR(store.GetArtifactsByContext(request, &response));
+        num_context_edges += response.artifacts_size();
+        break;
+      }
+      case FillContextEdgesConfig::ASSOCIATION: {
+        GetExecutionsByContextRequest request;
+        request.set_context_id(context.id());
+        GetExecutionsByContextResponse response;
+        TF_RETURN_IF_ERROR(store.GetExecutionsByContext(request, &response));
+        num_context_edges += response.executions_size();
+        break;
+      }
+      default:
+        LOG(FATAL) << "Wrong specification for FillContextEdges!";
+    }
+  }
+  return tensorflow::Status::OK();
+}
+
+// Inserts `num_attributions` attributions and `num_associations` associations
+// inside db. Returns detailed error if query executions failed.
+tensorflow::Status InsertContextEdgesInDb(const int64 num_attributions,
```

Comment:
drop?

Reply:
Still keep this function but set up and run another FillContextEdges workload inside as we discussed. Thanks! :)